kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/ProxyReconcilerIT.java

```diff
@@ -134,19 +138,16 @@ CreatedResources doCreate() {
 
     private void assertDeploymentBecomesReady(KafkaProxy proxy) {
         // wait longer for initial operator image download
-        await().alias("Deployment as expected").untilAsserted(() -> {
+        AWAIT.alias("Deployment as expected").untilAsserted(() -> {
             var deployment = extension.get(Deployment.class, ProxyDeployment.deploymentName(proxy));
-            assertThat(deployment).isNotNull()
-                    .extracting(Deployment::getStatus)
+            assertThat(deployment)
                     .describedAs("All deployment replicas should become ready")
-                    .satisfies(status -> {
-                        assertThat(status.getReplicas()).isEqualTo(status.getReadyReplicas());
-                    });
+                    .returns(true, Readiness::isDeploymentReady);
```

Comment:
This isn't actually required for the fix, but as @robobario, by using the utility method, we can hide the complexity.